controllers/binding/snapshotenvironmentbinding_controller.go

```diff
@@ -32,6 +34,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// AppStudioSnapshotEnvironmentBindingConditionsFailed is the condition to check for failures within the
+	// Snapshot Environment BindingConditions status
+	AppStudioSnapshotEnvironmentBindingConditionsFailed string = "False"
```

Comment:
Apologies, the example that I shared in the story had the value set as `False` when it should have been `True`

Reply:
ack, updated in most recent patch.